### ISSUE TITLE
Which Nvidia drivers work best?

### ISSUE THREAD

**Issue by Granville646** [NebuTech/NBMiner#650]
Hi All, I am mostly mining Ergo on a 3080ti. I have been using old drivers for some time without issue. I think that was 472.11

I upgraded to 496.76 and my mining performance has dropped by about a third from 256mh/s to 80mh/s. I dont need to be on the newest driver, but would rather be a bit more current. Can anyone recommend a known working version of the Nvidia drivers?

I have tried a number of LHR options, but im getting poor results as well. On 472.11 I didnt need to apply any LHR modes to get perfect performance.

Thanks

**Comment by Gorskor** [comment]
Roll it back my man, and see if it comes back. Easiest answer given you omit any mention of an issue forcing you to use a newer version.

**Comment by Granville646** [comment]
> Roll it back my man, and see if it comes back. Easiest answer given you omit any mention of an issue forcing you to use a newer version.

yeah, I rolled it back already and its fine. I just wonder what version people are using that might be newer.

**Comment by IordX** [comment]
try to remove driver with Display Driver Uninstaller , disable windows auto driver insatll in the app settings.
and install 496.76 again , I guess it will fix the problem
